lib/calculate.rb

```diff
@@ -0,0 +1,7 @@
+module Calculate
```

Comment:
I'm not sure if a `Calculate` module was necessary? As an order system does have a responsibility to tell you the total. Think about an invoice – it's the invoices responsibility to show a total based on the billed items. I would have had a `#total` inside `Order`.

Thoughts?